recipes/tcllib/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "tcllib" %}
+{% set version = "1.20" %}
+{% set sha256 = "e3b097475bcb93c4439df4a088daa59592e1937beee2a2c8495f4f0303125d71" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://core.tcl-lang.org/tcllib/uv/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "tclsh installer.tcl -no-wait -no-gui -no-html -no-nroff -no-examples"
+
+requirements:
+  host:
+    - tk
+  run:
+    - tk
+
+test:
+  commands:
+    - tclsh $RECIPE_DIR/logtofile.tcl  # [not win]
+    - tclsh %RECIPE_DIR%\logtofile.tcl  # [win]
```

Comment:
Don't use `RECIPE_DIR` env variable in test section. See `test: files` section

Reply:
Good point, I'll fix that!